<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/login_background.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>20</y>
      <width>308</width>
      <height>370</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;
}
QStackedWidget{
border:sold 10px rgba(0,0,0);
}
QWidget{
border:sold 10px rgba(0,0,0);
background-image: url(:/images/login_background.jpg);
}
QLineEdit{
	background-image: url(:/images/block.jpg);
	color: rgb(0, 0, 0);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton{
background:rgb(255, 255, 255);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
	font: 20pt &quot;宋体&quot;;
}
QPushButton:pressed{
background-color:#ced1d8;
border-style:inset;
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}
QLabel{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
font-size:14px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="login_page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_know">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>91</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Remember</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(0, 0, 255);
text-decoration: underline;
font: 24pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>华光彩云_CNKI</family>
        <pointsize>30</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 56pt &quot;小米兰亭&quot;;
font: 30pt &quot;华光彩云_CNKI&quot;;</string>
      </property>
      <property name="text">
       <string>File Server</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Registration">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:15px;
color: rgb(255, 255, 0);
font-style:MingLiU-ExtB;
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>Registration</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reg_page">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>用户注册</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>昵称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>邮箱：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_username">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_nicheng">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_password">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_queren">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_phone">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="reg_email">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: transparent;

font: 12pt &quot;宋体&quot;;
text-decoration: underline;

color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>注册提交</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="back_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qrc.qrc">
        <normaloff>:/images/back.jpeg</normaloff>:/images/back.jpeg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="set_page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>88</x>
        <y>51</y>
        <width>135</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>服务器设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>121</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="setpage_svip">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>121</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>192.168.1.1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="setpage_sport">
      <property name="geometry">
       <rect>
        <x>128</x>
        <y>172</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="placeholderText">
       <string>80</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>171</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>服务器端口：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: transparent;

font: 12pt &quot;宋体&quot;;
text-decoration: underline;

color: rgb(0, 0, 255);</string>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qrc.qrc">
        <normaloff>:/images/back.jpeg</normaloff>:/images/back.jpeg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="min">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/images/login_min.png</normaloff>:/images/login_min.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="close">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/images/login_close.png</normaloff>:/images/login_close.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="set">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="qrc.qrc">
      <normaloff>:/images/login_setting.png</normaloff>:/images/login_setting.png</iconset>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_id</tabstop>
  <tabstop>lineEdit_pw</tabstop>
  <tabstop>checkBox_know</tabstop>
  <tabstop>Registration</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>set</tabstop>
  <tabstop>min</tabstop>
  <tabstop>close</tabstop>
  <tabstop>reg_username</tabstop>
  <tabstop>reg_nicheng</tabstop>
  <tabstop>reg_password</tabstop>
  <tabstop>reg_queren</tabstop>
  <tabstop>reg_phone</tabstop>
  <tabstop>reg_email</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>setpage_svip</tabstop>
  <tabstop>setpage_sport</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
